### DISCUSSION TITLE
1.9b2

### DISCUSSION THREAD

**Discussion by sgillies** [Toblerity/Fiona#1183] [Announcements]
1.9b2 (2023-01-22)
------------------
The second 1.9 beta release is ready for early testing. All of the features planned for 1.9.0 are in this release.

The binary wheels on PyPI include GDAL 3.5.3, GEOS 3.11.1, and PROJ 9.0.1. To try the wheels, run pip install --pre fiona.

#### Changes

- Add `Feature.__geo_interface__ property` (#1181).
- Invalid creation options are filtered and ignored (#1180).
- The readme doc has been shortened and freshened up, with a modern example for version 1.9.0 (#1174). Advanced installation instructions have been moved to a separate doc.
- The Geometry class now provides and looks for `__geo_interface__` (#1174).
- The top level fiona module now exports Feature, Geometry, and Properties (#1174).
- Functions that take Feature or Geometry objects will continue to take dicts or objects that provide `__geo_interface__` (#1177). This reverses the deprecation introduced in 1.9a2.
- Python ignores SIGPIPE by default. By never catching BrokenPipeError via `except Exception` when, for example, piping the output of rio-shapes to the Unix head program, we avoid getting an unhandled BrokenPipeError message when the interpreter shuts down (#2689).

<hr /><em>This discussion was created from the release <a href='https://github.com/Toblerity/Fiona/releases/tag/1.9b2'>1.9b2</a>.</em>

**Comment by sgillies** [comment]
If no show stoppers are found, 1.9.0 will be released Monday, 2023-01-30.

**Comment by jorisvandenbossche** [comment]
With those latest changes (https://github.com/Toblerity/Fiona/pull/1177), the geopandas test suite now runs without changes without warnings with this latest beta. Thanks!
